react-native-reanimated.d.ts

```diff
@@ -476,6 +476,7 @@ declare module 'react-native-reanimated' {
       pageX: number;
       pageY: number;
     };
+    export function scrollTo(ref: RefOject<ReactNativeScrollView>, x: number, y: number, animated: boolean);
```

Comment:
I believe you have a typo here `RefOject` -> `RefObject`

Reply:
Thks a lot, that helped me fixed another issue with it. It seems to work great now.

Reply:
done, thank you :)